### ISSUE TITLE
Refactor hide/show code cells

### ISSUE THREAD

**Issue by mathieuboudreau** [Notebook-Factory/BayesianRepetoireOverlap#2]
Please do in a branch or fork then make a pull request.

## To do:

* [x] Remove text next to code cell buttons
  * [x] Easiest way, just regenerate book.
* [x] Remove CSS code that we added to make spaces between cell buttons [see here](https://github.com/Notebook-Factory/BayesianRepetoireOverlap/commit/d83c3ef569730ae170201836d1094b3c54d7c50f).
* [x] Remove code that created cell buttons (cell tag inside jupyter notebooks)
* [x] Create button that does nothing.
  *  [x] Create a copy of button html (e.g. https://github.com/Notebook-Factory/BayesianRepetoireOverlap/blob/master/_includes/buttons/thebelab.html)
  * [x] add it to [buttons.html](https://github.com/Notebook-Factory/BayesianRepetoireOverlap/blob/master/_includes/buttons.html)
  * [x] Add condition to the config: e.g. https://github.com/Notebook-Factory/BayesianRepetoireOverlap/blob/444e334807af809a9a45f9480bdf455475e9e602/_config.yml#L67
* [ ] Add event listener to the button that toggles the code cells.
  * [ ] Inspire from this: https://github.com/Notebook-Factory/BayesianRepetoireOverlap/blob/master/_includes/js/thebelab-cell-button.html
    * [ ] Should toggle the code cells (look inside html to see what should be toggled, likely just `.input`)
    * [ ] Inspire from javascript template code (e.g. try this one first: https://stackoverflow.com/questions/7183448/javascript-expand-collapse-multiple-divs-with-simple-code, then if not try this one https://www.w3schools.com/howto/howto_js_collapsible.asp)
  * [ ] Add listener to bottom of html (before closing body tag), but will look like this: https://github.com/Notebook-Factory/BayesianRepetoireOverlap/blob/444e334807af809a9a45f9480bdf455475e9e602/_layouts/default.html#L5-L6

**Comment by mathieuboudreau** [comment]
@zelenkastiot @FilipVel What's the status of the button functionality? Are you stuck on a point? If so, please let @TommyBoshkovski know ASAP.

**Comment by zelenkastiot** [comment]
Hey guys, sorry for the late reply. During the office hours, we got it to this stage: https://github.com/zelenkastiot/BayesianRepetoireOverlap

I haven't worked on it yet these days, I will get right to it!

**Comment by FilipVel** [comment]
Hey, I will be seeing it tomorrow morning.  As @agahkarakuzu  suggested at the end of the meeting on Tuesday, I'll try to figure this out by implementing a simple button that shows a pop up message, rather than trying to mimic the thebelab button. I'll see where it goes.

**Comment by mathieuboudreau** [comment]
Why is mimicking the button not possible - that was always the plan as far as i was aware? As always, if you are stuck on creating the button, please just advice from @TommyBoshkovski so that he can help.

**Comment by FilipVel** [comment]
When we were trying on tuesday, there was an error in console when calling the onClick function. I'll try to recreate what we've done then, and if the error persists I will be able to explain in detail so that @TommyBoshkovski can help.

**Comment by zelenkastiot** [comment]
I've got nothing on this matter so if Filip does not come up with something we should reach out to Tommy. I tried out but it does not seem to work for me. 

I will try the https://github.com/Notebook-Factory/BayesianRepetoireOverlap/issues/3 by today's office hours.

**Comment by zelenkastiot** [comment]
Sorry for closing the issue by accident. I tried working on the functionality of the button but it didn't seem to work for me.
If Filip doesn't succeed by today as well we should reach out to Tommy. 

I will work on the https://github.com/Notebook-Factory/BayesianRepetoireOverlap/issues/3 issue by today's office hours, I may seem to get some example of what we discussed.

**Comment by FilipVel** [comment]
So I got the button working properly by implementing a function in the `print.html` file.
The function code is:
```javascript
showhide = () => {
  var els = document.getElementsByClassName("input");
  Array.prototype.forEach.call(els, function(el) {
     if (el.style.display === "none") {
            el.style.display = "block";
    }
     else {
        el.style.display = "none";

  }

});

};
```

The function is being declared as a listener to the newly added button in the `initPrint` method which is being invoked by the framework by default.


I also set the `{% include js/code-cell-button.html %}` line to be in `_layouts/default.html` instead of  `_includes/js/thebelab.html`

**Comment by mathieuboudreau** [comment]
Resolved in #4
